backend/infrahub/graphql/types/attribute.py

```diff
@@ -131,6 +131,25 @@ class Meta:
         interfaces = {AttributeInterface}
 
 
+class MacAddressType(BaseAttribute):
+    value = Field(String)
+    oui = Field(String)
+    ei = Field(String)
+    version = Field(Int)
+    binary = Field(String)
+    eui48 = Field(String)
+    eui64 = Field(String)
+    bare = Field(String)
+    dot_notation = Field(String)
+    semicolon_notation = Field(String)
+    split_notation = Field(String)
```

Comment:
We haven't used it in a lot of places before but the Field object also takes a `description` parameter which would make it show up in the GraphQL schema. Might be good to describe the formats, such as "classic Cisco style" etc.